components/jvm-build-service/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+resources:
+- https://github.com/redhat-appstudio/jvm-build-service/deploy/kcp?ref=bc1ecfaf82f1edaef5b4d80e53d093e371048fb7
```

Comment:
no longer need to manage the system-config / operator-images items that were present in the pre-kcp branch